### PR TITLE
Adds a new ship, the NC Loader.

### PR DESCRIPTION
**Pull request by SurfinNinja1**
## About the PR

A new cargo ship, the NC Loader, has finished development and is ready to be rolled out to Frontier Outpost!  With a tentative price of 16,000 spesos, this compact cargo carrier has all the necessary necessities needed for anyone who wishes to haul freight for profit.  Amenities include a conveyor belt assembly next to the docking airlocks (I can smell the spaghetti from here!), a moderately sized cargo hold with room for 10 crates(even more if you want to go beyond the catwalks!), and a break room with an arcade machine and soda dispenser.
## Why / Balance

I like doing solo cargo runs and noticed there were no ships made for that, so I decided just to build one myself.  I also had never built a ship before so this was a fun learning experience (save for me messing up files and having to rebuild it TWICE).

## Technical details

I followed the youtube guide. There are no code changes here.
## Media
NC Loader

- [ ✔️ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
add: NC Loader

### CODE REVIEW

**Comment by Everturning** [comment]
It seems a bit too similar to the Chisel, which is bigger. I see the arcade machine as the only real difference, with a lot of aspects being the same (three drinks, PACMAN generator, captain's locker) and the main attraction being the cheap price.

**Comment by YumiGumi** [comment]
> It seems a bit too similar to the Chisel, which is bigger. I see the arcade machine as the only real difference, with a lot of aspects being the same (three drinks, PACMAN generator, captain's locker) and the main attraction being the cheap price.

this is a cargo shuttle, not a mining shuttle, doesn't even have an ore processor

**Comment by SurfinNinja1** [comment]
> It seems a bit too similar to the Chisel, which is bigger. I see the arcade machine as the only real difference, with a lot of aspects being the same (three drinks, PACMAN generator, captain's locker) and the main attraction being the cheap price.

As Yumi pointed out, this vessel is meant for cargo, not mining.  PACMAN generators are common to most small and medium ships, as are captain's lockers.  As for the 3 glasses, I only had 3 there because people in the discord thought that 2 was not enough.

**Comment by Cheackraze** [comment]
this is a super cute little cargo truck. Like a spess pickup truck. I like it.

File: Resources/Maps/Shuttles/Loader.yml
Comment (Cheackraze) [inline Resources/Maps/Shuttles/Loader.yml:1]: lowercase names only please

**Review by Cheackraze — CHANGES_REQUESTED** [review]

File: Resources/Maps/Shuttles/loader.yml [hunk 1]
```diff
@@ -0,0 +1,4226 @@
...
+          - 0
+          - 0
+          - 0
+          - 0
+          - 0
+        chunkSize: 4
+      type: GridAtmosphere
+    - type: GasTileOverlay
+    - type: RadiationGridResistance
+    - id: loader
```
Comment (Cheackraze) [inline Resources/Maps/Shuttles/loader.yml:1173]: this doesnt match properly

**Pull request merged**